tests/system/test_fim/test_synchronization/test_synchronization.py

```diff
@@ -0,0 +1,154 @@
+"""
+copyright: Copyright (C) 2015-2021, Wazuh Inc.
+           Created by Wazuh, Inc. <info@wazuh.com>.
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+type: system
+brief: Check that when FIM is activated, and the agent is running, the agent and manager are synchronization when
+a change is performed in a monitored folder.
+tier: 1
+modules:
+    - fim
+components:
+    - manager
+    - agent
+path: tests/system/test_fim/test_fim_synchronization/test_fim_synchronization.py
+daemons:
+    - wazuh-syscheckd
+os_platform:
+    - linux
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - CentOS 6
+    - Ubuntu Focal
+    - Ubuntu Bionic
+    - Ubuntu Xenial
+    - Ubuntu Trusty
+    - Debian Buster
+    - Debian Stretch
+    - Debian Jessie
+    - Debian Wheezy
+    - Red Hat 8
+    - Red Hat 7
+    - Red Hat 6
+references:
+    - https://github.com/wazuh/wazuh-qa/issues/2434
+tags:
+    - fim_synchronization
+"""
+
+import os
+import json
+import pytest
+from time import sleep
+
+
+from wazuh_testing.tools.monitoring import HostMonitor
+from wazuh_testing.tools.system import HostManager
+from test_fim import create_folder_file, clean_logs, query_db
+
+
+# Hosts
+testinfra_hosts = ["wazuh-manager", "wazuh-agent1"]
+
+inventory_path = os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))),
+                              'provisioning', 'one_manager_agent', 'inventory.yml')
+host_manager = HostManager(inventory_path)
+local_path = os.path.dirname(os.path.abspath(__file__))
+messages_path = [os.path.join(local_path, 'data/messages.yml'),
+                 os.path.join(local_path, 'data/delete_message.yml'),
+                 os.path.join(local_path, 'data/wait_fim_scan.yml'),
+                 os.path.join(local_path, 'data/agent_initializing_synchronization.yml'),
+                 os.path.join(local_path, 'data/manager_initializing_synchronization.yml')
+                 ]
+tmp_path = os.path.join(local_path, 'tmp')
+scheduled_mode = 'testdir1'
+db_path = '/var/ossec/queue/db/001.db'
+db_script = '/var/system_query_db.py'
+
+
+@pytest.mark.parametrize('host', ['wazuh-agent1', 'wazuh-manager'])
+@pytest.mark.parametrize('case', ['add', 'modify', 'delete'])
+@pytest.mark.parametrize('folder_path', ['testdir1'])
+def test_Synchronization_create_file_agent_stopped(folder_path, case, host):
```

Comment:
The name should be representative. It doesn't just create a file (it also modifies and deletes it) and, the description says that it is when the manager or agent stops (2 cases). However, the test name only talks about agent stopped.

Reply:
Done in commit [7ae07](https://github.com/wazuh/wazuh-qa/pull/2536/commits/7ae0720f1da0c910bc5ba364481db18d525d6294)